src/App.tsx

```diff
@@ -1,61 +1,132 @@
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { FormEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const preparedTodos = () => {
+    return todosFromServer.map(todo => {
+      const user = usersFromServer.find(
```

Comment:
You use usersFromServer.find(...) at leat twice (check the line 66 in this file). It is a dublicating of code. You can create a new folder 'utilities', create a file like 'CreateUser' and write a function with this logic. And after that import this function at the line 12 and 66

Reply:
> You use usersFromServer.find(...) at leat twice (check the line 66 in this file). It is a dublicating of code. You can create a new folder 'utilities', create a file like 'CreateUser' and write a function with this logic. And after that import this function at the line 12 and 66

Good point
Also, `prepareTodos` can be just a variable outside the component as you need to do it once